Formula/ipfs.rb

```diff
@@ -32,8 +32,7 @@ def install
     system "make", "build"
     bin.install "cmd/ipfs/ipfs"
 
-    bash_output = Utils.safe_popen_read(bin/"ipfs", "commands", "completion", "bash")
-    (bash_completion/"ipfs-completion.bash").write bash_output
+    generate_completions_from_executable(bin/"ipfs", "commands", "completion", shells: [:bash])
```

Comment:
Do we need this?

```suggestion
    generate_completions_from_executable(bin/"ipfs", "commands", "completion", base_name: "ipfs-completion", shells: [:bash])
```

Reply:
No, the installed completion works without the change.

Reply:
How did you test this?

Reply:
I installed the formula from source from this PR and checked (in bash) if the completions where working.
(Note: if you want to reproduce this, the system bash on Monterey is too old for the completion script, you'll need brewed bash.)